src/components/footer.css

```diff
@@ -0,0 +1,10 @@
+/* stylelint-disable */
+.footer{
+    background-color: rgb(11, 25, 87);
+    text-align: center;
+    padding: 10px;
+    margin: 0;
+    font-size: 20px;
+    
+
+}
```

Comment:
- [ ] Kindly add those lines to make the footer at the bottom of the page `position: absolute;`, `width: 100vw;`, and `bottom: 0;`.

Before | After
--------- | -------
![Screenshot from 2022-08-02 01-43-27](https://user-images.githubusercontent.com/33752735/182263096-00a0240d-5861-4f24-9c14-91040ea91a4b.png) | ![Screenshot from 2022-08-02 01-43-11](https://user-images.githubusercontent.com/33752735/182263106-762e3f82-c2bb-404b-9313-41a7d1dc37c0.png)